packages/block/src/block.ts

```diff
@@ -230,37 +231,50 @@ export class Block {
    * @returns the block specified by `blockTag`
    */
   public static fromEthersProvider = async (
-    provider: ethers.providers.JsonRpcProvider | string,
+    provider: any,
     blockTag: string | bigint,
     opts: BlockOptions
   ) => {
     let blockData
-    const prov =
-      typeof provider === 'string' ? new ethers.providers.JsonRpcProvider(provider) : provider
+    const providerUrl = getProvider(provider)
```

Comment:
so the provider is essentially being stripped out. not sure if this is the right approach as the provider might be providing some wrapper functionality (like proof seeking and verification)

May be we can later revisit this on how to best interface a provider

Reply:
Where would a provider do proof verification?  We could put cleanly define the provider interface (at least for the minimal functions we'd need to call) I supposed but that feels like more work than it's worth and very brittle if `ethers` or `web3js` (if we support that provider type too in the future) ever change their API.  At the end of the day, we're just making an HTTP call to some remote RPC provider to get the json blob back for the blob or the tx.  But, that said, happy to discuss further.  We need to add a todo on the breaking release to rename the `fromEthersProvider` functions anyway.

Reply:
Can we eventually test for the Ethers import to work (in a try/catch block) and only do this stripping out as a fallback solution? Then we might somewhat have best of both worlds.

Reply:
I guess we could but all we were doing before was calling the `provider.send` method which is just abstracting a raw JSON RPC call.  Here again, we weren't using any provider-specific functionality, just a simple wrapper around HTTP requests.  As such, I'm not sure what we gain since we'd have to blindly call the provider and hope nothing in the API has changed.  I don't think what I've done in this PR is any worse than that.

Reply:
Ok, good reasoning, then let's merge here.

I would now plan a round of "normal" releases for sometime mid next week, will prepare release notes next Monday or so.